tensorflow/core/platform/env.h

```diff
@@ -218,6 +218,10 @@ class Env {
   /// Creates a local unique temporary file name. Returns true if success.
   bool LocalTempFilename(string* filename);
 
+  /// Creates a local unique file name that starts with |prefix| and ends with
+  /// |suffix|. Returns true if success.
+  bool CreateLocalFilename(string* prefix, const string& suffix);
```

Comment:
After pondering the implementation, I think perhaps CreateNewFileWithUniqueName might be a better name for this function.

Reply:
Done